Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -258,6 +260,25 @@ - (void)removeChatWithAllMessages:(OCTChat *)chat
     });
 }
 
+- (void)removeAllCalls
```

Comment:
It would be better to rename method to something more explicit like `convertAllCallsToMessages`